sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/com/azure/resourcemanager/appservice/implementation/Utils.java

```diff
@@ -111,54 +105,16 @@ private static String removeTrailingChar(String s, char c) {
      * @return an Observable pointing to the content of the file
      */
     static Mono<byte[]> downloadFileAsync(String url, HttpPipeline httpPipeline) {
```

Comment:
On switch to `HttpPipeline`, this is the only code affected (rest is sample or test code).